versions/3.0.md

```diff
@@ -2214,8 +2215,8 @@ definitions:
 A simple object to allow referencing other definitions in the specification.
 It can be used to reference parameters and responses that are defined at the top level for reuse.
 
-The Reference Object is a [JSON Reference](http://tools.ietf.org/html/draft-pbryan-zyp-json-ref-02) that uses a [JSON Pointer](http://tools.ietf.org/html/rfc6901) as its value.
-For this specification, only [canonical dereferencing](http://json-schema.org/latest/json-schema-core.html#anchor27) is supported.
+The Reference Object is a [JSON Reference](https://tools.ietf.org/html/draft-pbryan-zyp-json-ref-03) that uses a [JSON Pointer](http://tools.ietf.org/html/rfc6901) as its value.
```

Comment:
That's what I was hoping for.  So in that case, the _"that uses a JSON Pointer as its value"_ is not accurate is it?  The _JSON Pointer_ version of a `$ref` value is for _local only_ and since a _Reference Object_ is for local and remote references, isn't the _Reference Object_'s `$ref` just a URI per the [JSON References spec](https://tools.ietf.org/html/draft-pbryan-zyp-json-ref-03#section-3).